source/administration/object-management/object-versioning.rst

```diff
@@ -149,12 +147,12 @@ uses a ``null`` version ID. You can access or remove these objects by specifying
 Versioned Delete Operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Performing a ``DELETE`` operation on a versioned object creates a 
-0-byte ``DeleteMarker`` as the latest version of that object. Clients performing
-``GET`` operations on that object do not return any results, as MinIO does not
-return the ``DeleteMarker`` back as part of the response. Similarly, performing
-a ``LIST`` operation by default returns only objects which are *not* a
-``DeleteMarker``.
+Performing a ``DELETE`` operation on a versioned object creates a 0-byte ``DeleteMarker`` as the latest version of that object.
+For objects where the latest version is a ``DeleteMarker``, clients must specify versioning flags or identifiers to ``GET/HEAD/LIST/DELETE`` operations to work on any version of that object.
+The default server behavior is to omit ``DeleteMarker`` objects from consideration for unversioned operations.
+
+MinIO relies on :ref:`Lifecycle Management expiration rules <minio-lifecycle-management-expiration>` or manual cleanup via versioned ``DELETE`` operations for removing versioned objects, including ``DeleteMarkers``. 
+MinIO does not otherwise automatically remove any versioned object.
```

Comment:
Perhaps drop `otherwise`?
```suggestion
MinIO does not automatically remove any versioned object.
```

Reply:
Given the previous sentence, is there any way a user might somehow arrive at the conclusion that we do not support automatic removal of versioned objects? Maybe

```
MinIO relies on Lifecycle Management expiration rules for automatic cleanup *or* versioned delete operations for manual cleanup of non-current versioned objects and ``DeleteMarker`` objects.
```

?

Reply:
Oh! I think I missed the point. Yes, you are correct. Nevermind. 😄

Reply:
I mean I like the new sentence better anyways so the feedback was still good